website/docs/cloud-docs/workspaces/dynamic-provider-credentials/hcp-configuration.mdx

```diff
@@ -0,0 +1,110 @@
+|-----------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------|
+| `TFC_HCP_WORKLOAD_IDENTITY_AUDIENCE`          | Will be used as the `aud` claim for the identity token. Defaults to the provider resource name for the current run phase that was derived from the values you provided for `RUN_PROVIDER_RESOURCE_NAME`, `PLAN_PROVIDER_RESOURCE_NAME`, and `APPLY_PROVIDER_RESOURCE_NAME` | Requires **v1.15.1** or later if self-managing agents. This is one of the default `aud` formats that HCP accepts.                          |
+| `TFC_HCP_PLAN_PROVIDER_RESOURCE_NAME`         | The resource name of the workload identity provider that will be used to assume the service principal for the plan phase of a run.                                                                                                                                         | Requires **v1.15.1** or later if self-managing agents. Will fall back to the value of `RUN_PROVIDER_RESOURCE_NAME` if not provided.        |
+| `TFC_HCP_APPLY_PROVIDER_RESOURCE_NAME`        | The resource name of the workload identity provider that will be used to assume the service principal for the apply phase of a run.                                                                                                                                        | Requires **v1.15.1** or later if self-managing agents. Will fall back to the value of `RUN_PROVIDER_RESOURCE_NAME` if not provided.        |
+
+## Configure the HCP Provider
+
+Make sure that you’re not setting values for the `HCP_CRED_FILE`environment variables as these will conflict with the dynamic credentials authentication process.
+
+### Specifying Multiple Configurations
+
+~> **Important:** If you are self-hosting [Terraform Cloud agents](/terraform/cloud-docs/agents), ensure your agents use [v1.15.1](/terraform/cloud-docs/agents/changelog#1-15-1-05-01-2024) or above. To use the latest dynamic credentials features, [upgrade your agents to the latest version](/terraform/cloud-docs/agents/changelog).
+
+You can add additional configurations to handle multiple distinct HCP setups, enabling you to use multiple [provider aliases](/terraform/language/providers/configuration#alias-multiple-provider-configurations) within the same workspace.
+
+For more details, see [Specifying Multiple Configurations](/terraform/cloud-docs/workspaces/dynamic-provider-credentials/specifying-multiple-configurations).
```

Comment:
"Refer to" is more accessible language for those reading with screen readers:
```suggestion
For more details, refer to [Specifying Multiple Configurations](/terraform/cloud-docs/workspaces/dynamic-provider-credentials/specifying-multiple-configurations).
```

Reply:
TIL!